nextflow_schema.json

```
@@ -842,14 +842,16 @@
                 },
                 "busco_db": {
                     "type": "string",
-                    "description": "Download URL for BUSCO lineage dataset, or path to a tar.gz archive, or local directory containing already downloaded and unpacked lineage datasets.",
-                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz or '/path/to/buscodb' (files still need to be unpacked manually). Available databases are listed here: https://busco-data.ezlab.org/v5/data/lineages/.",
+                    "description": "Download URL, tar.gz archive local path, or local uncompressed directory for an *_odb10 or *_odb12 BUSCO lineage datasets.",
+                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz, `/path/to/bacteria_odb10.2024-01-08.tar.gz`.\n\nFor more details, check the [BUSCO section in the usage documentation](https://nf-co.re/mag/latest/docs/usage/#busco).",
                     "format": "path",
                     "exists": true
                 },
-                "busco_auto_lineage_prok": {
-                    "type": "boolean",
-                    "description": "Run BUSCO with automated lineage selection, but ignoring eukaryotes (saves runtime)."
+                "busco_db_lineage": {
+                    "type": "string",
+                    "description": "Name of the BUSCO *_odb10 or *_odb12 lineage to check against. Additionally supports 'auto' and 'auto_prok' for automatic lineage selection mode.",
```

```suggestion
                    "description": "Name of the BUSCO *_odb10 or *_odb12 lineage to check against. Additionally supports 'auto', 'auto_prok', and 'auto_euk' for automatic lineage selection mode.",
```

Since we have Eukaryote-specific stuff in the pipeline